test/Elastica/ResponseTest.php

```diff
@@ -2,82 +2,15 @@
 
 namespace Elastica\Test;
 
-use Elastica\Document;
-use Elastica\Exception\ResponseException;
-use Elastica\Mapping;
-use Elastica\Query;
-use Elastica\Query\MatchAll;
-use Elastica\Request;
 use Elastica\Response;
 use Elastica\Test\Base as BaseTest;
 
+/**
+ * @group unit
+ */
 class ResponseTest extends BaseTest
 {
-    /**
-     * @group functional
-     */
-    public function testResponse()
```

Comment:
Ah, the other one is replacing this part?

Reply:
yes, exactly. :)